---
 debian/rules.d/build.mk |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: b/debian/rules.d/build.mk
===================================================================
--- a/debian/rules.d/build.mk
+++ b/debian/rules.d/build.mk
@@ -261,6 +261,7 @@
 	  esac; \
 	fi
 
+ifneq (,$(DEB_HOST_MULTIARCH))
 	# Create the ld.so symlink to the multiarch directory
 	if [ $(curpass) = libc ]; then \
 	  rtld_so="$$(LANG=C LC_ALL=C readelf -l debian/tmp-$(curpass)/usr/bin/iconv | grep 'interpreter' | sed -e 's/.*interpreter: \(.*\)]/\1/g')" ; \
@@ -269,7 +270,8 @@
 	  target="$(call xx,slibdir)/$$(readlink debian/tmp-$(curpass)/$(call xx,slibdir)/$$rtld_so)" ; \
 	  ln -s $$target $$link_name ;  \
 	fi
-	
+endif
+
 	$(call xx,extra_install)
 endif
 	touch $@
